feat: improved getting-started section (#732)
ansys/pyansys-geometry: doc/source/getting_started/local/index.rst

.. _ref_creating_local_session:

> Create a local session

Reviewer: I like the simple verb heading titles better!

> For more information on the arguments accepted by the launcher methods, please refer

Reviewer: ```suggestion
For more information on the arguments accepted by the launcher methods, see
```

> to their API documentation:

Reviewer: ```suggestion
their API documentation:
```

* `launch_modeler_with_discovery <../../api/ansys/geometry/core/connection/launcher/index.html#launcher.launch_modeler_with_discovery>`_
* `launch_modeler_with_spaceclaim <../../api/ansys/geometry/core/connection/launcher/index.html#launcher.launch_modeler_with_spaceclaim>`_
* `launch_modeler_with_geometry_service <../../api/ansys/geometry/core/connection/launcher/index.html#launcher.launch_modeler_with_geometry_service>`_

.. note::

    Because this is the first release of the Geometry service, you cannot yet define a product version
    or API version.

.. button-ref:: ../index
    :ref-type: doc
    :color: primary
    :shadow:
    :expand:

>     Go back to Getting Started

Reviewer: ```suggestion
    Go back to Getting started
```
See--here no italics.
